### ISSUE TITLE
FFHQ Custom class finetuning

### ISSUE THREAD

**Issue by leg0m4n** [phizaz/diffae#7]
Hi Konpat and Diffae team,

Thank you so much for your work! I am trying to create a new class for the head pose(actually 3: pitch, yaw, roll) and finetune it on FFHQ model. It's not a binary classification, so I assume d2c method of having positives and negatives won't work(I have concrete values for each), I need to switch the training loss function from binary cross-entropy to mse. Do I understand correctly that I need to finetune ffhq256_autoenc/latent.ckpt checkpoint? Is it the checkpoint of the encoder?

Thanks,
Richard
